index.html

```diff
@@ -48,8 +48,9 @@
         </header>
 
         <div>
-            <img src="https://ilawyer.com.sg/wp-content/uploads/2020/06/Depositphotos_67457429_l-2015-e1591177273833.jpg" width="600">
-            <p>Before you can begin resolving the conflict, you must wait until both of you are ready. Once you feel that you are calm and able to maintain an honest communication, it is time.</p>
+            <img src="https://images.unsplash.com/photo-1543852786-1cf6624b9987?q=80&w=1974&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D" width="600">
```

Comment:
Please change the width to 650. 
```suggestion
            <img src="https://images.unsplash.com/photo-1543852786-1cf6624b9987?q=80&w=1974&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D" width="650">
```

Reply:
Made this change